src/interfaces/business.ts

```diff
@@ -1,27 +1,28 @@
-export interface Business {
-  natureOfBusiness: string
-}
-
-export interface NameRequest {
```

Comment:
This interface and the one above weren't used anywhere else so there was no need to export them.

I could have left them un-exported for use in BusinessRequest below, but I didn't like them appearing to represent a Business (or NameRequest) but not containing all the necessary fields, so I just moved them into BusinessRequest below (which I think is more clear).